### PR TITLE
Disable updater + add feature gate

### PR DESCRIPTION
**Pull request by Brendonovich**
Hopefully resolves the release build issues
https://github.com/spacedriveapp/spacedrive/actions/runs/6015255188

### CODE REVIEW

File: packages/client/src/core.ts [hunk 1]
```diff
@@ -239,7 +239,7 @@ export type MaybeNot<T> = T | { not: T }
 
 export type MaybeUndefined<T> = null | null | T
 
-export type MediaData = { id: number; pixel_width: number | null; pixel_height: number | null; longitude: number | null; latitude: number | null; fps: number | null; capture_device_make: string | null; capture_device_model: string | null; capture_device_software: string | null; duration_seconds: number | null; codecs: string | null; streams: number | null }
+export type MediaData = { id: number; dimensions: number[] | null; media_date: number[] | null; media_location: number[] | null; camera_data: number[] | null; artist: string | null; description: string | null; copyright: string | null; exif_version: string | null; object_id: number }
```
Comment (HeavenVolkoff) [inline packages/client/src/core.ts:242]: This is an old version of `core.ts`

File: apps/desktop/src-tauri/scripts/tauri.js [hunk 2]
```diff
@@ -12,145 +12,138 @@ const [_, __, ...args] = process.argv;
...
+        );
+    })
+    .finally(() => {
+        for (const file of toRemove)
+            try {
+                fs.unlinkSync(file);
+            } catch (e) { }
+
+        process.exit(code);
+    });
```
Comment (HeavenVolkoff) [inline apps/desktop/src-tauri/scripts/tauri.js:149]: We are currently using tabs for all files, no? Should `js` files use spaces instead?

**Review by HeavenVolkoff — CHANGES_REQUESTED** [review]

File: apps/desktop/src-tauri/scripts/tauri.js (hunk 2 above)
Comment (Brendonovich) [reply apps/desktop/src-tauri/scripts/tauri.js]: misconfigured editor moment

**Review by HeavenVolkoff — APPROVED** [review]
LGTM

**Pull request merged**